### PR TITLE
users mvc updated

### PR DESCRIPTION
**Pull request by elywelly**
Wanted to check if my route names are right. Let me know if anything missing. Thanks!

### CODE REVIEW

File: controllers/users/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+const router = express.Router();
+
+router.get('/:id', (req, res) => {
+    Users.getById(req.params.id).then((user) => {
+        res.json(user);
+    });
+});
+
+router.get('/getByUsername', (req, res) => {
```
Comment (wprk) [inline controllers/users/index.js:12]: I'd recommend changing these slightly as it doesn't follow REST best practices. You would want the route to be `/` but then have a query parameter instead that switches the query type.

`/?username=joe.bloggs`
`/?email=joe.bloggs@gmail.com`

Try these instead

File: controllers/users/index.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    Users.getByUsername(req.body.username).then((username) => {
+        res.json(username);
+    });
+});
+
+router.get('/getByEmail', (req, res) => {
```
Comment (wprk) [inline controllers/users/index.js:18]: See above comment.

File: server.js [hunk 3]
```diff
@@ -11,9 +12,10 @@ const app = express();
 //     res.send('hello');
 // });
 
-app.use(express.static("client"));
+app.use(express.static('client'));
 
-app.use("/api/paymentEvents/", paymentEventsController);
+app.use('/api/paymentEvents/', paymentEventsController);
```
Comment (wprk) [inline server.js:17]: Just a personal preference but I prefer all lower-case and hyphenated routes.

**Review by wprk — APPROVED** [review]

File: controllers/users/index.js (hunk 1 above)
Comment (elywelly) [reply controllers/users/index.js:12]: Thanks! That was exactly what I was wondering/missing

**Review by kdaya001 — APPROVED** [review]

**Review by May-Manmai — APPROVED** [review]

**Review by ky-ra1 — APPROVED** [review]
👍🏻

**Pull request merged**